src/SchemaGenerator.ts

```diff
@@ -151,6 +151,9 @@ export class SchemaGenerator {
                     return;
                 }
                 return;
+            case ts.SyntaxKind.ArrowFunction:
```

Comment:
What about Functions, not just arrow functions?

Reply:
You're right, I missed this one. Will fix!

Reply:
Fixed in 464023647